code/modules/GOAP/monkey/info_provider.dm

```diff
@@ -0,0 +1,74 @@
+/datum/goap_info_provider/monkey/GetWorldState(datum/goap_agent/agent)
+	. = list()
+	.["GUN"] = FALSE
+	.["attackEnemy"] = FALSE
+	.["disposeEnemy"] = FALSE
+	.["Flee"] = FALSE
```

Comment:
it irks me you're not using consistent casing on these.

Reply:
Can fiz